Server/src/main/java/org/openas2/cmd/processor/restapi/ApiResource.java

```diff
@@ -220,6 +223,64 @@ public Response headCommand(@PathParam("param") String command) {
         return Response.status(200).build();
     }
 
+    @GET
+    @RolesAllowed({"ADMIN"})
+    @Path("/getPropertyList")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getPropertyList() {
+        Map<String, String> result = new HashMap<>();
+        result = (Map<String,String>) Properties.getProperties();
+
+        ObjectMapper om = new ObjectMapper();
+        try {
+            String js = om.writeValueAsString(result);
+            return Response.ok(js, MediaType.APPLICATION_JSON).build();
+        } catch (JsonProcessingException e) {
+             return Response.status(Response.Status.INTERNAL_SERVER_ERROR).entity("error").type(MediaType.APPLICATION_JSON).build();
+        }
+    }
+
+    @GET
+    @RolesAllowed({"ADMIN"})
+    @Path("/getXml")
+    @Produces(MediaType.APPLICATION_XML)
+    public Response getXml(@QueryParam("filename") String filename, @QueryParam("xpath") String xpathExpression) {
+        Session session = getProcessor().getSession();
+        String filePath = session.getBaseDirectory() + '\\' + filename;
```

Comment:
You must use a forward slash and leave Java to make the change if you run on Windows which it will normally gracefully handle for you or use File.SEPARATOR